examples/tutorials/starting/overview.py

```diff
@@ -481,7 +483,7 @@
 #
 # -  To learn more about those classes, go to the API docs (links are in
 #    the introduction at the top).
-# -  To learn more about other parts of Gammapy (e.g. Fermi-LAT and TeV
+# -  To learn more about other parts of Gammapy (e.g.Fermi-LAT and TeV
```

Comment:
The initial comment was correct, with a space...

Reply:
There was a weird character (not visible here) which I removed but forgot to put the space back in that case. I will correct.